tests/testthat/test-checkers.R

```diff
@@ -0,0 +1,42 @@
+test_that("is_parameterised works as expected for epiparam", {
+  res <- is_parameterised(epiparam())
+  expect_length(res, 118)
+  expect_type(res, type = "logical")
+})
+
+test_that("is_parameterised works as expected with epiparam parameters", {
+  expect_true(is_parameterised(epiparam()[15, ]))
```

Comment:
Again suggest not hardcoding examples as their position might change

Reply:
Resolved in 240c42f882ed9063c87b3e2b00f5f20e1cda0159 (see above).